documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-deploying-cruise-control-{context}']
+= Deploying Cruise Control
+
+Deploy Cruise Control through the `Kafka` resource in the `cruiseControl` properties.
+
+Refer to the xref:ref-sample-kafka-resource-config-deployment-configuration-kafka[sample Kafka YAML configuration] for an overview of the `Kafka` resource and its properties.
+
+The properties relevant to the Cruise Control configuration are shown in this procedure.
+
+You can configure these properties as part of a deployment or redeployment of the Kafka cluster.
+
+.Prerequisites
+
+* A Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `cruiseControl` properties for the `Kafka` resource.
```

Comment:
Aren't we editing the `cruiseControl` "property" singular? Makes more sense to say "edit the configuration options under the `cruiseControl` property of the `kafka` resource. 

Also I am not sure "property" is the correct word (unless this is K8s specific term).

Reply:
I updated to the singular "property". There are two things I am trying to keep distinct: the Cruise Control Deployment configuration, `Kafka.cruiseControl`, and the Cruise Control application configuration, `Kafka.cruiseControl.config`. I would like to refer to `Kafka.cruiseControl.config` as a property for Cruise Control configuration